docs/reference/airnode/next/concepts/gas-prices.md

```diff
@@ -133,6 +144,54 @@ to`type 0` and a `gasPrice` value.
 provider reported gas price. The resulting Gas Price will equal
 `Gas Price * providerRecommendedGasPrice`.
 
+## sanitizedProviderRecommendedGasPrice
+
+The `sanitizedProviderRecommendedGasPrice` strategy builds upon `providerRecommendedGasPrice` strategy by leveraging Base Fee reported in block header also ensuring that the gas price remains reasonable and capped based on specified parameters when necessary. The strategy calculates the gas estimation by using the `providerRecommendedGasPrice`. It then compares this value to the Base Fee multiplied by the baseFeeMultiplierThreshold. If the multiplied gas estimation exceeds the threshold, it multiplies the Base Fee with the baseFeeMultiplier, adds the priorityFee, and returns this value. Otherwise, it returns the value from the `providerRecommendedGasPrice` strategy. Similar to the former, it sets the transaction to`type 0` and a `gasPrice` value.
```

Comment:
Clarifying suggestion
```suggestion
The `sanitizedProviderRecommendedGasPrice` strategy builds upon the `providerRecommendedGasPrice` strategy to ensure that the gas price remains reasonable and capped based on specified parameters. The strategy estimates gas by first determining the `providerRecommendedGasPrice`. It then compares this value, multiplied by `recommendedGasPriceMultiplier`, to the Base Fee reported in block headers multiplied by the `baseFeeMultiplierThreshold`. If the former is greater than the latter, it multiplies the Base Fee with the `baseFeeMultiplier`, adds the `priorityFee`, and returns this value. Otherwise, it returns the value from the `providerRecommendedGasPrice` strategy. Similar to the former, it sets the transaction to `type 0` and a `gasPrice` value.
```

Reply:
Your version is much better. Thanks for that 🙏🏼